recipes/omniscidb-cuda/recipe/activate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export CONDA_OMNISCIDB_BACKUP_PATH=$PATH
+export PATH=$CONDA_PREFIX/opt/omnisci/bin:$PATH
+
+ln -s $CONDA_PREFIX/opt/omnisci/bin/initdb $CONDA_PREFIX/bin/omnisci_initdb
```

Comment:
Can you do this in the recipe?

Reply:
yes.
But what is your aim here? Eliminate activate/deactivate scripts?

Reply:
> But what is your aim here? Eliminate activate/deactivate scripts?

Yes.

Reply:
The activate scripts are eliminated.